docs/contributor/draft/rate-limiting-poc/06-sum-up-rate-limiting.md

```diff
@@ -0,0 +1,36 @@
+# Summary of Rate Limiting POC
+
+Istio supports rate limiting by using EnvoyFilter configurations to handle the rate limiting in the Envoy proxies. This can be set to local or global rate limiting. Local rate limiting does not require communication with a rate limit service, and each Envoy instance acts independently. Global rate limiting requires gRPC communication with a rate limit service, for example [envoy ratelimit service](https://github.com/envoyproxy/ratelimit).
+
+Envoy rate limit service needs to use Redis or Memcached backend.
+
+# Memcached
+
+Pros:
+- Mature project, used in production by many
+- Easy to deploy in a Kyma cluster
+Cons:
+- Experimental support in Envoy
+- Key size limitation of 250 characters
+- Rate limiting based on the client's certificate or very long header values is not possible
+
+# Redis
+
+Pros:
+- Mature and popular
+- Supported by Envoy
+- Some know-how within the Goat team
+Cons:
+- Licence issues
```

Comment:
Maybe we can add a bit more information, not everyone might be aware what those issues are.